src/Compilers/Test/Utilities/CSharp/SymbolUtilities.cs

```diff
@@ -120,9 +120,10 @@ public static string[] ToTestDisplayStrings(this IEnumerable<ISymbol> symbols)
             return symbols.Select(s => s.ToTestDisplayString()).ToArray();
         }
 
-        public static string[] ToTestDisplayStrings(this IEnumerable<Symbol> symbols)
+        public static string[] ToTestDisplayStrings(this IEnumerable<Symbol> symbols, SymbolDisplayFormat format = null)
         {
-            return symbols.Select(s => s.ToTestDisplayString()).ToArray();
+            format ??= SymbolDisplayFormat.TestFormat;
```

Comment:
nit: consider doing something like the following here:

```cs
            return symbols.SelectAsArray(static (s, format) => s.ToDisplayString(format), format);
```

Reply:
I'll leave as-is to get this into preview3. Thanks